girder/api/v1/group.py

```diff
@@ -204,59 +178,46 @@ def joinGroup(self, group, params):
         return group
 
     @access.public
-    @loadmodel(model='group', level=AccessType.READ)
     @filtermodel(model='user')
-    @describeRoute(
+    @autoDescribeRoute(
         Description('List members of a group.')
-        .param('id', 'The ID of the group.', paramType='path')
+        .modelParam('id', model='group', level=AccessType.READ)
         .pagingParams(defaultSort='lastName')
         .errorResponse('ID was invalid.')
         .errorResponse('Read access was denied for the group.', 403)
     )
-    def listMembers(self, group, params):
-        limit, offset, sort = self.getPagingParameters(params, 'lastName')
-
-        return list(self.model('group').listMembers(
-            group, offset=offset, limit=limit, sort=sort))
+    def listMembers(self, group, limit, offset, sort, params):
+        return list(self.model('group').listMembers(group, offset=offset, limit=limit, sort=sort))
 
     @access.user
-    @loadmodel(model='group', level=AccessType.WRITE)
     @filtermodel(model='group', addFields={'access', 'requests'})
-    @describeRoute(
-        Description("Invite a user to join a group, or accept a user's request "
-                    " to join.")
+    @autoDescribeRoute(
+        Description("Invite a user to join a group, or accept a user's request to join.")
         .responseClass('Group')
         .notes('The "force" option to this endpoint is only available to '
                'administrators and can be used to bypass the invitation process'
                ' and instead add the user directly to the group.')
-        .param('id', 'The ID of the group.', paramType='path')
-        .param('userId', 'The ID of the user to invite or accept.')
-        .param('level', 'The access level the user will be given when they '
-               'accept the invitation.',
-               required=False, dataType='int', default=AccessType.READ)
+        .modelParam('id', model='group', level=AccessType.WRITE)
+        .modelParam('userId', 'The ID of the user to invite or accept.',
+                    destName='userToInvite', level=AccessType.READ, paramType='formData')
```

Comment:
Users have always been ACModels, we just don't expose a lot of control over their ACLs at this point.